ui/browsing/browsingpane.go

```diff
@@ -274,3 +279,13 @@ func (b *BrowsingPane) CurrentPage() controller.Route {
 func (b *BrowsingPane) CreateRenderer() fyne.WidgetRenderer {
 	return widget.NewSimpleRenderer(b.container)
 }
+
+func (b *BrowsingPane) updateNavBtnsColor(p Page) {
+	for pageName, icon := range b.navBtnsPageMap {
+		if pageName == p.Route().Page {
+			icon.(*theme.ThemedResource).ColorName = theme.ColorNamePrimary
+		} else {
+			icon.(*theme.ThemedResource).ColorName = ""
```

Comment:
The default color name is `theme.ColorNameForeground`. We should use that instead of empty string here

Reply:
Thanks! I changed the color name to use the default